_posts/newsletter/2021-07-08-this-month-in-solid.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: newsletter
+title: This Month in Solid 2021-05-06
+permalink: /newsletter/2021-05-06
+tags: [monthly, updates]
+categories: [Updates]
+---
+
+## Updates
+
+### [Events](https://solidproject.org/events)
+
+**1 July 2021** **— Solid World July —** Academics were at the forefront of July’s Solid World. There were many exciting and collaborative pieces coming together this month — from hearing the winner of a Solid challenge to learning how students are innovating new versions of applications.
```

Comment:
Maybe "announcing" or "finding out"  instead of "hearing"? Somehow, "hearing the winner" has a somewhat dampened effect (but it could totally be my own personal quirks - so feel free to ignore).